torch/_inductor/scheduler.py

```diff
@@ -1048,25 +1054,6 @@ def compute_dependencies(self):
         """
         name_to_users = collections.defaultdict(list)
 
-        # handle aliasing by using python aliasing in name_to_users
```

Comment:
Remove this part since it looks like some sort of dead code. Before this loop, name_to_users is an empty dict. After this loop, all the values in name_to_users should be empty list. Which does not add much extra benefits here.

Reply:
NVM, this code is very important...